pkg/nseconfig/validate.go

```diff
@@ -46,6 +47,14 @@ func (v VL3) validate() error {
 		}
 	}
 
+	if pl, err := strconv.Atoi(v.IPAM.PrefixLength); err == nil {
```

Comment:
after prefixlength will be int this will be missing, also you should check if `err != nil` almost all the time

Reply:
done